### DISCUSSION TITLE
F-22 AoA Oscillation starting at Mach 1.1

### DISCUSSION THREAD

**Discussion by Stupefy** [JSBSim-Team/jsbsim#629] [Q&A]
Using MIXR and trying to use the F-22 model I notice that the angle of attack and g load starts oscillating at mach 1.1 and above.  The velocity vector stays constant.  Any hints as to what may be causing this?

**Comment by seanmcleod** [comment]
What is MIXR?

**Reply by Stupefy** [reply]
I'm sorry, MIXR is "OpenEAAGLES" rebranded.  It is a simulation framework that include JSBSim as an example aerodynamics model.

**Comment by seanmcleod** [comment]
Have you tried logging some of the parameters like control inputs to the FCS and the output of the FCS to the control surfaces?

**Reply by Stupefy** [reply]
[fileOutput.csv](https://github.com/JSBSim-Team/jsbsim/files/8808161/fileOutput.csv)

**Reply by Stupefy** [reply]
I started the sim right when the oscillations begin.  unsure what parameters I am looking for that I can control.

**Comment by seanmcleod** [comment]
Can you show us a plot of things like alpha, g, elevator just before the oscillations start and for a little while during the oscillations?

**Reply by Stupefy** [reply]
![image](https://user-images.githubusercontent.com/5407714/171263873-640d64c7-cae0-412e-85ce-dd2e18673715.png)
![image](https://user-images.githubusercontent.com/5407714/171263902-b4085c08-9eba-4d38-8ced-0a401d4ea6a6.png)
![image](https://user-images.githubusercontent.com/5407714/171264105-351b830d-a9d5-4a8c-9f63-bdfabc2f04ce.png)

**Reply by agodemar** [reply]
It doesn't look weird to me. The elevator seems the cause of all this. But we should look at all the plots to figure out the problem.
What are the units of measure of the y-axes?
What's the set of initial conditions?
Where's the F-22 model you are using?

**Comment by Stupefy** [comment]
I did and all control surfaces are at zero.  Landing gear output starts at
1 and moves to zero at start.  Is there a logger function I can enable and
provide the output of here?

**Reply by seanmcleod** [reply]
But your graphs posted just above show the elevator position moving. Are you talking about a different setup now?
